src/backend/core/api/permissions.py

```diff
@@ -52,3 +54,23 @@ def has_object_permission(self, request, view, obj):
         """Check permission for a given object."""
         abilities = obj.get_abilities(request.user)
         return abilities.get(request.method.lower(), False)
+
+
+class RelatedTeamAccessPermission(IsAuthenticated):
```

Comment:
I looked into it and now I think we shouldn't merge this PR without PR https://github.com/numerique-gouv/people/pull/65.
In fact this `invitation` endpoint should work exactly like the `team-access` endpoint. The filtering of accesses related to a team can be done the `get_queryset` method without incurring extra queries. See these lines in the TeamAccessViewset:
```

    def get_queryset(self):
        """Return the queryset according to the action."""
        queryset = super().get_queryset()
        queryset = queryset.filter(team=self.kwargs["team_id"])
```

Reply:
OK. What I was suggesting the other day was to 
- merge this PR here quickly to allow Anto to work with this creating endpoint
- keep on advancing on #65 to complete the CRUD soon and handle listing / deletion soon

I can modify this PR to make it match team accesses but I still think we should merge it quickly and improve the finer details regarding the whole endpoint on next PR.